setup.py

```diff
@@ -35,26 +30,39 @@ def get_install_requires():
 install_requires, dependency_links = get_install_requires()
 
 
+def get_long_description():
+    with open('README.rst', 'r') as f:
+        return f.read()
```

Comment:
You can do the same with the `read_file()` method added above. I don't see what necessitates the need for either of these changes though.